reaction_components/grabbable.js

```diff
@@ -74,9 +75,13 @@ AFRAME.registerComponent('grabbable', inherit(base, {
       return
     }
     // room for more grabbers?
-    const grabAvailable = !Number.isFinite(this.data.maxGrabbers) ||
+    let grabAvailable = !Number.isFinite(this.data.maxGrabbers) ||
         this.grabbers.length < this.data.maxGrabbers
-
+    if (Number.isFinite(this.data.maxGrabbers) && !grabAvailable &&
+        this.grabbed && this.data.maxGrabBehavior === 'drop') {
+      this.grabbers[0].components['super-hands'].onGrabEndButton()
```

Comment:
This would crash the component if it has button filtering enabled with a TypeError from trying to access properties on a null event detail.  
https://github.com/wmurphyrd/aframe-super-hands-component/blob/master/reaction_components/prototypes/buttons-proto.js#L5